Flow.Launcher.Plugin.Everything.csproj

```diff
@@ -118,14 +118,14 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Flow.Launcher.Plugin" Version="1.4.0" />
     <PackageReference Include="Droplex" Version="1.0.4" />
-    <PackageReference Include="Flow.Launcher.Plugin" Version="1.3.1" />
-    <PackageReference Include="JetBrains.Annotations" Version="2019.1.3" />
+	<PackageReference Include="JetBrains.Annotations" Version="2019.1.3" />
     <PackageReference Include="System.Runtime" Version="4.3.1" />
   </ItemGroup>
 
   <ItemGroup>
-    <Reference Include="OLD_Flow.Launcher.Infrastructure">
+    <Reference Include="Flow.Launcher.Infrastructure">
```

Comment:
is this still needed?

Reply:
There are still quite a lot of code that needs this.
1. The JSONStorage (which is not supposed to be in intrastracture)
2. ISavable interface
3. Log
4. maybe some more

Reply:
I do agree that we shall gradually move this need to the api